tests/phpunit/tests/test-admin-filters-post.php

```diff
@@ -20,6 +20,8 @@ public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
 		self::create_language( 'es_ES' );
 
 		self::$editor = $factory->user->create( array( 'role' => 'editor' ) );
+
+		require_once POLYLANG_DIR . '/include/api.php';
```

Comment:
Strange on my side only `$GLOBALS['polylang']` assignment is needed

Reply:
Indeed because the API is already loaded here https://github.com/polylang/polylang/blob/3.6.5/tests/phpunit/includes/testcase-assets.php#L2.